pegasus/sites.v3/code.org/public/ops-dashboard/index.haml

```diff
@@ -70,11 +71,11 @@ title: "Ops Dashboard"
   services.factory('programTypesService', [
     function() {
       var program_types = {
-        1: {id: 1, short_name: 'CS in Science', long_name: 'Computer Science in Science'},
-        2: {id: 2, short_name: 'CS in Algebra', long_name: 'Computer Science in Algebra'},
-        3: {id: 3, short_name: 'ECS', long_name: 'Exploring Computer Science'},
-        4: {id: 4, short_name: 'CSP', long_name: 'Computer Science Principles'},
-        5: {id: 5, short_name: 'K5', long_name: 'Computer Science in K5'}
+        1: {id: 1, short_name: "#{ ActivityConstants::PROGRAM_TYPES[1][:short_name]}", long_name: "#{ ActivityConstants::PROGRAM_TYPES[1][:long_name]}"},
```

Comment:
Would it make sense to use a loop here that iterates over all PROGRAM_TYPES?  That way the code would just work if we added a new program type.